Gopkg.toml

```diff
@@ -76,3 +76,6 @@ ignored = ["github.com/improbable-eng/thanos/benchmark/*"]
   name = "github.com/minio/minio-go"
   version = "6.0.1"
 
+[[constraint]]
+  name = "github.com/Azure/azure-storage-blob-go"
```

Comment:
We need to pin it to some version, for sure - we don't want reproducible builds.

Reply:
not addressed

Reply:
Pinned latest commit, because version tag is a bit outdated and don't even have API version which I use for this project